### DISCUSSION TITLE
Number of concept images for training a specific style?

### DISCUSSION THREAD

**Discussion by jojoha1337** [TheLastBen/fast-stable-diffusion#1039] [Q&A] [answered]
Just finished training a model finetuned from the Waifu Diffusion 1.3 model ( the style i want to train is also anime-esque ).
Its nailing the style pretty well except that it doesn't quite get how the style draws the faces and their facial expressions ( it works pretty well at replicating how the style renders the bodies of the characters but not how the style does the faces )

I used 20 instance images at their default settings (3000 training, 350 text encoder), all are mostly full body shots.
But i only used 44 concept images at 330 steps, i've seen that people were using around 100 or 200 of concept images for training but i decided to try for less to save time collecting more images.

Would increasing the amount of concept images improve the accuracy of how the model can render faces/facial expressions in the style i want or should i try something else?

**Comment by TheLastBen** [comment, marked as answer]
concept images help with diversity of the output, not with the quality of the faces, if you want good quality faces, make sure the instance pictures include faces from multiple perspectives. it's best to use +100 concept images and 1000 steps for the concept.
